code/modules/mob/freelook/ai/eye.dm

```diff
@@ -62,12 +62,12 @@
 	return ..()
 
 /atom/proc/move_camera_by_click()
-	if(istype(usr, /mob/living/silicon/ai))
-		var/mob/living/silicon/ai/AI = usr
-		if(AI.eyeobj && AI.client.eye == AI.eyeobj)
+	if(istype(usr, /mob/living/silicon/ai) || istype(usr, /mob/hacker))

```

Comment:
Not very object-oriented.

Reply:
I don't see much harm in allowing any eyeobj to be moved by clicking, but you might want to check if that causes problems anywhere.